.travis.yml

```diff
@@ -100,6 +105,26 @@ matrix:
               ':!*platform_mbed.h' ':!*TESTS/*' ':!TEST_APPS/' ':!UNITTESTS/' \
               ':!*tests/*' ':!*targets/*' ':!*TARGET_*' ':!*unsupported/*'
 
+    - &cmake-build-test
+      stage: "CMake"
+      name: "CMake blinky example k64f"
+      env: NAME=cmake_test
+      install:
+        - >-
+         git clone --depth=1 --single-branch --branch feature-cmake https://github.com:ARMmbed/mbed-os-example-blinky.git;
+         cd mbed-os-example-blinky && mkdir build && cd build
+      script:
+        - cmake .. -GNinja -DMBED_TOOLCHAIN=GCC_ARM -DMBED_TARGET=K64F && ninja
+
+    - <<: *cmake-build-test
+      name: "CMake storage example"
```

Comment:
Can the name also be explicit about the target as above?

Reply:
as I am fixing silly errors, I added building each example to multiple targets.